atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/schema/stream/generated/StreamTestTableFactory.java

```diff
@@ -38,73 +41,72 @@ public static StreamTestTableFactory of() {
+    public StreamTestWithHashStreamMetadataTable getStreamTestWithHashStreamMetadataTable(Transaction t,
+            StreamTestWithHashStreamMetadataTable.StreamTestWithHashStreamMetadataTrigger... triggers) {
         return StreamTestWithHashStreamMetadataTable.of(t, namespace, Triggers.getAllTriggers(t, sharedTriggers, triggers));
     }
 
-    public StreamTestWithHashStreamValueTable getStreamTestWithHashStreamValueTable(Transaction t, StreamTestWithHashStreamValueTable.StreamTestWithHashStreamValueTrigger... triggers) {
+    public StreamTestWithHashStreamValueTable getStreamTestWithHashStreamValueTable(Transaction t,
+            StreamTestWithHashStreamValueTable.StreamTestWithHashStreamValueTrigger... triggers) {
         return StreamTestWithHashStreamValueTable.of(t, namespace, Triggers.getAllTriggers(t, sharedTriggers, triggers));
     }
 
-    public interface SharedTriggers extends
-            KeyValueTable.KeyValueTrigger,
-            StreamTestMaxMemStreamHashAidxTable.StreamTestMaxMemStreamHashAidxTrigger,
-            StreamTestMaxMemStreamIdxTable.StreamTestMaxMemStreamIdxTrigger,
-            StreamTestMaxMemStreamMetadataTable.StreamTestMaxMemStreamMetadataTrigger,
-            StreamTestMaxMemStreamValueTable.StreamTestMaxMemStreamValueTrigger,
-            StreamTestStreamHashAidxTable.StreamTestStreamHashAidxTrigger,
-            StreamTestStreamIdxTable.StreamTestStreamIdxTrigger,
-            StreamTestStreamMetadataTable.StreamTestStreamMetadataTrigger,
-            StreamTestStreamValueTable.StreamTestStreamValueTrigger,
-            StreamTestWithHashStreamHashAidxTable.StreamTestWithHashStreamHashAidxTrigger,
-            StreamTestWithHashStreamIdxTable.StreamTestWithHashStreamIdxTrigger,
-            StreamTestWithHashStreamMetadataTable.StreamTestWithHashStreamMetadataTrigger,
-            StreamTestWithHashStreamValueTable.StreamTestWithHashStreamValueTrigger {
-        /* empty */
+    public interface SharedTriggers extends KeyValueTable.KeyValueTrigger, StreamTestMaxMemStreamHashAidxTable.StreamTestMaxMemStreamHashAidxTrigger, StreamTestMaxMemStreamIdxTable.StreamTestMaxMemStreamIdxTrigger, StreamTestMaxMemStreamMetadataTable.StreamTestMaxMemStreamMetadataTrigger, StreamTestMaxMemStreamValueTable.StreamTestMaxMemStreamValueTrigger, StreamTestStreamHashAidxTable.StreamTestStreamHashAidxTrigger, StreamTestStreamIdxTable.StreamTestStreamIdxTrigger, StreamTestStreamMetadataTable.StreamTestStreamMetadataTrigger, StreamTestStreamValueTable.StreamTestStreamValueTrigger, StreamTestWithHashStreamHashAidxTable.StreamTestWithHashStreamHashAidxTrigger, StreamTestWithHashStreamIdxTable.StreamTestWithHashStreamIdxTrigger, StreamTestWithHashStreamMetadataTable.StreamTestWithHashStreamMetadataTrigger, StreamTestWithHashStreamValueTable.StreamTestWithHashStreamValueTrigger {
```

Comment:
This is a very long line, likely to break baseline for the schema consuumers. Is there someway to specify the line length limit?

Reply:
There doesn't seem to be a Javapoet feature to specify a line length limit. But I'm expecting consumers to have a baseline exception for gen files, as we do.